Diagnostic/diagnostic.py

```diff
@@ -695,6 +695,26 @@ def start_daemon():
         hutil.error("wait daemon start time out")
 
 
+def check_suspected_memory_leak(pid):
+    memory_leak_threshold_in_KB = 2000000  # Roughly 2GB
```

Comment:
Using a hard-coded 2GB doesn't sound like a good idea. On some small VMs like A1, 2GB is a lot of memory. Can this be made as some config parameter somewhere?